all_repos/clone.py

```diff
@@ -27,7 +27,10 @@ def _get_current_state_helper(
     for direntry in os.scandir(path):
         if direntry.name == '.git':
             seen_git = True
-        elif direntry.is_dir():  # pragma: no branch (defensive)
+        elif (
+            direntry.is_dir() and
+            not direntry.name.startswith('.')
```

Comment:
please don't change this, you've run `sudo` in a directory you shouldn't have and that's user error and not something this tool should adjust for


also it's unrelated to your change